.storybook/preview.tsx

```diff
@@ -1,38 +1,95 @@
-        <div>
-            <ThemeProvider theme={theme}>
-                <CssBaseline />
+        <ThemeProvider theme={theme}>
+            <CssBaseline />
+            <GlobalStyles
+                styles={{
+                    html: { height: '100%', width: '100%' },
+                    body: {
+                        height: '100%',
+                        width: '100%',
+                        backgroundColor: themeKey === 'side-by-side' ? '#e4e4e4' : undefined,
+                    },
+                    '#storybook-root': { height: '100%', width: '100%', display: 'flex' },
+                }}
+            />
+
+            {themeKey === 'side-by-side' ? (
+                // If the theme is side-by-side, render the story twice in two different themes
+                <Box sx={{ display: 'flex', gap: '1rem', width: '100%', height: '100%' }}>
+                    <ThemeProvider theme={THEMES['light']}>
+                        <Box
+                            sx={(theme) => ({
+                                display: 'flex',
+                                flex: 1,
+                                backgroundColor: 'background.paper',
+                                overflow: 'auto',
+                                ...useNiceScrollbars(theme.palette.background.paper),
+                            })}
+                        >
+                            <Story />
+                        </Box>
+                    </ThemeProvider>
+                    <ThemeProvider theme={THEMES['dark']}>
+                        <Box
+                            sx={(theme) => ({
+                                display: 'flex',
+                                flex: 1,
+                                backgroundColor: theme.palette.background.paper,
+                                overflow: 'auto',
+                                ...useNiceScrollbars(theme.palette.background.paper),
+                            })}
+                        >
+                            <Story />
+                        </Box>
+                    </ThemeProvider>
+                </Box>
+            ) : (
                 <Story />
-            </ThemeProvider>
-        </div>
+            )}
+        </ThemeProvider>
     );
 };
 
+export const decorators = [withMuiTheme];
+
 const preview: Preview = {
-    decorators: [
-        (Story) => (
-            <div>
-                <Story />
-            </div>
-        ),
-    ],
+    globalTypes: {
+        theme: {
+            name: 'Theme',
+            title: 'Theme',
+            description: 'Theme for our components',
+            defaultValue: 'dark',
+            toolbar: {
+                icon: 'paintbrush',
+                dynamicTitle: true,
+                items: [
+                    { value: 'light', left: '☀️', title: 'Light mode' },
+                    { value: 'dark', left: '🌙', title: 'Dark mode' },
+                    { value: 'side-by-side', left: '🌗', title: 'Side by side' },
```

Comment:
Love this option!

Reply:
Thanks - speeds it up to see both at the same time :)